packages/Checkbox/src/Checkbox.styles.js

```diff
@@ -9,8 +9,17 @@ const getLabelLeftPadding = (checkboxSize, hasLabel) =>
 
 const smallCheckboxSize = tokens.checkbox.sizeSm;
 const mediumCheckboxSize = tokens.checkbox.sizeMd;
-const largeCheckboxSize = tokens.checkbox.sizeLg;
 
+const mediumCheckboxGeneralStyle = {
+  checkBoxStyles: {
+    height: mediumCheckboxSize,
+    width: mediumCheckboxSize,
+  },
```

Comment:
Propbably makes sense to match the style of the rest of this styling, but I would think it would make more sense to define these `styles` variables as CSS:

```js
checkboxStyles: css`
  height: ${mediumCheckboxSize};
  width: ${mediumCheckboxSize};
`
```